### ISSUE TITLE
New language : EO

### ISSUE THREAD

**Issue by pirboazo** [squidfunk/mkdocs-material#1883]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/3c2ox0C)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "eo",
  "direction": "ltr",
  "clipboard.copy": "Kopii al tondujo",
  "clipboard.copied": "Kopiado al klipo",
  "edit.link.title": "Redakti ĉi tiun paĝon",
  "footer.previous": "antaŭa",
  "footer.next": "sekva",
  "footer.title": "Piedlinio",
  "header.title": "kaplinio",
  "meta.comments": "komentoj",
  "meta.source": "Fontkodo",
  "nav.title": "Navigado",
  "search.config.lang": "eo",
  "search.config.pipeline": "tondetilo, haltigi vort-filtrilon",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "Serĉo",
  "search.reset": "klara",
  "search.result.placeholder": "Tajpu por komenci serĉadon",
  "search.result.none": "Neniuj kongruaj dokumentoj",
  "search.result.one": "1 kongrua dokumento",
  "search.result.other": "# kongruaj dokumentoj",
  "skip.link.title": "saltu al enhavo",
  "source.link.title": "iru al deponejo",
  "source.revision.date": "lasta ĝisdatigo",
  "tabs.title": "langetoj",
  "toc.title": "Enhavtabelo"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Added in 44882568.

**Comment by squidfunk** [comment]
Released as part of 5.5.9
